data/xml/1962.earlymt.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1962.earlymt">
+  <volume id="1" ingest-date="2021-10-30">
+    <meta>
+      <booktitle>Automatic translation of languages NATO Summer School</booktitle>
+      <publisher>Pergamon Press</publisher>
+      <address>Venice</address>
+      <month>July</month>
+      <year>1962</year>
+    </meta>
+    <frontmatter>
+      <url hash="6e5c2d0d">1962.earlymt-1.0</url>
+      <bibkey>earlymt-1962-automatic</bibkey>
+    </frontmatter>
+    <paper id="1">
```

Comment:
Actually it seems that all the papers are off by 1 (PDF 1 is the frontmatter, PDF 2 is paper 1, etc)

Reply:
I have re-uploaded the PDFs, should be fixed now.